charts/deepgram-self-hosted/templates/license-proxy/license-proxy.deployment.yaml

```diff
@@ -50,6 +50,34 @@ spec:
       {{- if or .Values.licenseProxy.serviceAccount.create .Values.licenseProxy.serviceAccount.name }}
       serviceAccountName: {{ default (printf "%s-sa" .Values.licenseProxy.namePrefix) .Values.licenseProxy.serviceAccount.name }}
       {{- end }}
+
+      volumes:
+        - name: configmap-volume
+          configMap:
+            name: {{ .Values.licenseProxy.namePrefix }}-config
+        - name: working-config
+          emptyDir: {}
+
+      initContainers:
+        - name: config-render
+          image: alpine:latest
+          command: ["/bin/sh"]
+          args:
+            - -c
+            - apk add --no-cache gettext-envsubst && envsubst < /config/license-proxy.toml > /work/license-proxy.toml
```

Comment:
I don't understand the adjustment here to dynamically inject the config instead of using the statically defined asset. Could you explain that for me?

Reply:
To maybe make that more obvious to future readers, could we add a comment to that effect, and maybe gate this approach to customers enabling `billing`, and leave the standard deploy flow as-is?